kong/plugins/request-transformer/handler.lua

```diff
@@ -1,15 +1,18 @@
+local BasePlugin = require "kong.plugins.base_plugin"
 local access = require "kong.plugins.request-transformer.access"
 
+local RequestTransformerHandler = BasePlugin:extend()
 
-local RequestTransformerHandler = {
-  VERSION  = "1.2.2",
-  PRIORITY = 801,
-}
-
+function RequestTransformerHandler:new()
+  RequestTransformerHandler.super.new(self, "request-transformer")
+end
 
 function RequestTransformerHandler:access(conf)
+  RequestTransformerHandler.super.access(self)
```

Comment:
Calling `super` like this is also not needed any more (you can remove this line)